test/wanda/catalog/checks_customizations_test.exs

```diff
@@ -1,4 +1,4 @@
-defmodule Wanda.ChecksCustomizationsTest do
+defmodule Wanda.Catalog.ChecksCustomizationsTest do
```

Comment:
The same. I think this should go in the `test/wanda` folder, as you are testing `wanda/checks_customizations.ex` file.